platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/status/ConfigurationStatus.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.status;
+
+import java.nio.file.Path;
+
+public class ConfigurationStatus {
```

Comment:
Instead of having setters, can we pass things in the constructor(s) and make the fields `final` and class immutable?

We should also add javadoc comments to clearly describe when the `message` and `path` are used for, what values they can have, etc.